spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb

```diff
@@ -50,10 +54,21 @@ def perform
     end
 
     def perform
-      resource.required_resource_attributes
+      resource.required_resource_attributes(include_id: include_id)
     end
 
     it { expect(perform).to eq(expected_attributes) }
+
+    context "with included id" do
+      let(:include_id) { true }
+      let(:expected_attributes) do
+        [
+          { name: :id, type: :integer, swagger_type: :integer, example: kind_of(Integer), required: false },
+          { name: :title, type: :string, swagger_type: :string, example: "'Some title'", required: true },
+          { name: :body, type: :text, swagger_type: :string, example: "'Some body'", required: true }
+        ]
+      end
+    end
```

Comment:
- This context doesn't have an example
- Shouldn't it be `required: true` in the `id` line?

Reply:
No, id column should be treated like special attribute